dojo-dungeon/src/dojo/utils/studentTasks/modifyEntities/ModifyEntities.java

```diff
@@ -0,0 +1,42 @@
+package dojo.utils.studentTasks.modifyEntities;
+
+import core.Entity;
+
+/** Class to modify game entities. */
```

Comment:
hier könnte noch ein satz hinzugefügt werden, dass diese die klasse ist, welche die studis in raum x in level y ergänzen sollen. dann wissen sie, dass sie richtig sind ...